packages/yarnpkg-pnpify/sources/NodePathResolver.ts

```diff
@@ -54,7 +64,7 @@ export interface ResolvedPath<PathType extends FSPath<PortablePath>> {
  * The idea: for path like `node_modules/foo/node_modules/bar` we use `foo` as an issuer
  * and resolve `bar` for this issuer using `pnpapi`.
  */
-export class NodePathResolver {
+export class NodePathResolver implements PathResolver {
```

Comment:
Why the interface? It seems to only be used in a single place, right?

Reply:
You are right, I can do without interface, just need to change `readDir` method to private scope.